libraries/xcode.rb

```diff
@@ -43,7 +43,7 @@ def installed_path
 
     def current_path
       if installed_path.nil? && @version.match?(/beta/i)
-        '/Applications/Xcode-beta.app'
+        "/Applications/Xcode-#{@apple_version}.Beta.app"
```

Comment:
Where does this naming come from? I get the previous change when I download the beta from the developer site. Just curious is all.